packages/client/rollup.config.js

```diff
@@ -20,6 +20,19 @@ const ignoredWarnings = [
   "a11y-click-events-have-key-events",
 ]
 
+const devPaths = production
+  ? []
+  : [
+      {
+        find: "@budibase/shared-core",
+        replacement: path.resolve("../shared-core/dist/mjs/src/index"),
```

Comment:
Is this still needed, even with both `shared-core` and `types` now specifying a default ESM `import` export? The client should be able to consume it now without these additional aliases, like it does for `string-templates` which also gets built to both CJS and ESM bundles.

Reply:
It's needed for the local dev for now, as the published package does not have it, and rollup does not know about the local package.

Reply:
This only runs on development, for production it will always read from the npm repo as it does now